src/FSharp.DependencyManager.Nuget/FSharp.DependencyManager.fs

```diff
@@ -24,12 +24,22 @@ module FSharpDependencyManager =
         | true, false -> v
         | _ -> ""
 
-    let validateAndFormatRestoreSources (sources: string) =
+    let validateAndFormatRestoreSources (directoryPath: string) (sources: string) =
         [|
             let items = sources.Split(';')
 
             for item in items do
-                let uri = Uri(item)
+                let uri =
+                    let mutable res: Uri = null
+                    if Uri.TryCreate(item, UriKind.Absolute, &res) then
```

Comment:
Can't something like Path.IsPathRooted used here instead (or other filesystem APIs). Is Uri checking the existence of the directory? I would personally expect it won't?

Reply:
`Path.IsPathRooted` does not work for other schemes, like https, while this seems to work fine for any scheme